tests/kickstarts/rhel7.cfg

```diff
@@ -0,0 +1,139 @@
+# SCAP Security Guide SCAP Test Suite node Red Hat Enterprise Linux 7 Server
```

Comment:
Good call, I have merged both kickstarts to one. I have named it `rhel_centos_7` but inside, it's only RHEL - obviously, CentOS being carbon copy, there is no need to have it commented inside.